app/util/path-builder.ts

```diff
@@ -66,6 +66,8 @@ export const pb = {
   vpcs: (params: Project) => `${pb.project(params)}/vpcs`,
   vpc: (params: Vpc) => `${pb.vpcs(params)}/${params.vpc}`,
   vpcEdit: (params: Vpc) => `${pb.vpc(params)}/edit`,
+  floatingIps: (params: Project) => `${pb.project(params)}/floating-ips`,
+  floatingIpsNew: (params: Project) => `${pb.project(params)}/floating-ips-new`,
```

Comment:
For whatever reason (I am probably to blame) the convention for this name is `floatingIpNew`, even though we always put the `s` in the path. We should probably rename all of these to match the paths (separate PR obviously).